dashboard.el

```diff
@@ -180,6 +163,78 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
     (beginning-of-line)
     (widget-forward 1)))
 
+(defun dashboard-generate-banner ()
```

Comment:
Banner is not a selectable widget, it should always be there so i would remove it from the list of widgets